src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -171,6 +174,16 @@ namespace winrt::TerminalApp::implementation
             }
         });
 
+        newTabImpl->ExportTabRequested([weakTab, weakThis{ get_weak() }]() {
+            auto page{ weakThis.get() };
+            auto tab{ weakTab.get() };
+
+            if (page && tab)
+            {
+                page->_ExportTab(*tab);

```

Comment:
an aside: we may want to (in a future PR) turn this into an action, that could let the user export the pane to a file from the command palette too, not just the context menu. But now I'm thinking of all the other kinds of actions we could have for this feature...